_posts/2019-02-05-barcode.html

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title:  "Barcode"
+date:   2019-02-05 09:30:30 +0100
+categories: Experiment
+teaser: Experiment
+excerpt: >
+  Creating art out of headline images
+---
+<p><a href="https://github.com/ftlabs/barcode" target="_blank">Barcode</a> is an internal service/endpoint which gathers all of the headline images used in FT articles from a provided date range, the last 24 hours for example, and squashes them to give one condensed image. The final result looks similar to a coloured barcode.</p>
+
+{% include figure.html src="/assets/uploads/2019/01/barcode_header_example.jpg" description="Parameters used to generate this image: width=1024, height=768, dateFrom=2018-11-13, timeFrom=00:00:00, dateTo=2018-11-14, timeTo=00:00:00, orientation=v, fit=fill, order=colour, sort=asc" width="80%" %}
```

Comment:
I would say this is unneeded detail in the img description. Clearer would be "Barcode of articles published during 2018-11-13, sorted by colour"

Reply:
Ok, amended